app/app.js

```diff
@@ -15,7 +15,7 @@ const appViews = [
   configPaths.layouts,
   configPaths.partials,
   configPaths.examples,
-  configPaths.components
+  configPaths.govukFrontend
```

Comment:
Is there a reason this is not 'components' anymore? It'd allow easier reuse if we don't tie this code to govukFrontend?